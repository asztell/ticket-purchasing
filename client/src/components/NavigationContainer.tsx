import classnames from 'classnames'
import './NavigationContainer.scss'

export function NavigationContainer({
  className,
  children
}: {
  className?: string
  children: React.ReactNode
}): JSX.Element {
  const combinedClassName = classnames('Navigation-Container', className)
  return <div className={combinedClassName}>{children}</div>
}
