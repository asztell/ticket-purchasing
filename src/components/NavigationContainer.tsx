import classnames from "classnames";
import "./NavigationContainer.scss";

export function NavigationContainer({
  className,
  children,
}: {
  className?: string;
  children: React.ReactNode;
}) {
  className = classnames("Navigation-Container", className);
  return <div className={className}>{children}</div>;
}
