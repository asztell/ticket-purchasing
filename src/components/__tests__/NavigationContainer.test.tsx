import { render } from "@testing-library/react";
import { NavigationContainer } from "../NavigationContainer";

describe("<NavigationContainer />", () => {
  test("render NavigationContainer", () => {
    const { container } = render(
      <NavigationContainer>test</NavigationContainer>
    );
    expect(container).toMatchSnapshot();
  });
});
