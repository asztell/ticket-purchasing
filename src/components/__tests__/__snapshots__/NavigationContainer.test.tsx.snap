// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationContainer /> render NavigationContainer 1`] = `
<div>
  <div
    class="Navigation-Container"
  >
    test
  </div>
</div>
`;
